simulation_parameters/Constants.cs

```diff
@@ -1356,11 +1356,11 @@ public static class Constants
 
     public const float TIME_BEFORE_TUTORIAL_CAN_PAUSE = 0.01f;
 
-    public const float MICROBE_MOVEMENT_EXPLAIN_TUTORIAL_DELAY = 12.0f;
+    public const float MICROBE_MOVEMENT_EXPLAIN_TUTORIAL_DELAY = 3.5f;
     public const float MICROBE_MOVEMENT_EXPLAIN_TUTORIAL_DELAY_CONTROLLER = 1.0f;
-    public const float MICROBE_MOVEMENT_TUTORIAL_REQUIRE_DIRECTION_PRESS_TIME = 2.2f;
+    public const float MICROBE_MOVEMENT_TUTORIAL_REQUIRE_DIRECTION_PRESS_TIME = 1.6f;
     public const float TUTORIAL_ENTITY_POSITION_UPDATE_INTERVAL = 0.2f;
-    public const float GLUCOSE_TUTORIAL_TRIGGER_ENABLE_FREE_STORAGE_SPACE = 0.14f;
+    public const float GLUCOSE_TUTORIAL_TRIGGER_ENABLE_FREE_STORAGE_SPACE = 0.03f;
```

Comment:
Doesn't this cause problems with the player being able to find glucose way too soon? I've balanced the tutorial so that at most you need to spend like one second cluelessly in the glucose cloud wondering if it is working or not. With less storage space the player may need to spend a lot longer inside a cloud attempting to absorb it even when their storage is full.

Reply:
This variable could be put back probably given the other tutorials, but there's a new change later to set the glucose manually so that there's always enough free space to finish that tutorial phase.

Reply:
Yeah, I did see that code later. To me it doesn't seem exactly good that the player data is passed to the tutorial like that to be modified, that's part of the major architectural problem I think exists here I commented on in my review.